### PR TITLE
Kkraune/speedtest

### PR DESCRIPTION
**Pull request by kkraune**
- only two last commits relevant for review

### CODE REVIEW

File: en/vespa-feed-client.html [hunk 1]
```diff
@@ -28,11 +28,17 @@ <h3>Java library</h3>
...
   <li>Install <a href="build-install-vespa.html"><em>vespa-clients</em>/<em>vespa</em></a> RPM package.</li>
+  <li>Download <a href="https://search.maven.org/artifact/com.yahoo.vespa/vespa-feed-client-cli">
+    vespa-feed-client <strong>zip</strong> artifact</a> from Maven Central.</li>
 </ul>
-
+<p>Download example:</p>
+<pre>
+$ F_REPO="https://repo1.maven.org/maven2/com/yahoo/vespa/vespa-feed-client-cli" && \
+  F_VER=$(curl -Ss "${F_REPO}/maven-metadata.xml" | sed -n 's/.*&lt;release&gt;\(.*\)<.*>/\1/p') && \
+  curl -SsLo vespa-feed-client-cli.zip ${F_REPO}/${F_VER}/vespa-feed-client-cli-${F_VER}-zip.zip && \
```
Comment (jonmv) [inline en/vespa-feed-client.html:39]: 🌟

**Review by jonmv — APPROVED** [review]
Thanks!

File: en/vespa-feed-client.html (hunk 1 above)
Comment (kkraune) [reply en/vespa-feed-client.html:39]: this is @bjorncs magic in action!

**Review by bjorncs — COMMENTED** [review]
👍

File: en/vespa-feed-client.html [hunk 2]
```diff
@@ -28,11 +28,17 @@ <h3>Java library</h3>
...
-    vespa-feed-client <strong>zip</strong> artifact</a> from Maven Central.
   <li>Install <a href="build-install-vespa.html"><em>vespa-clients</em>/<em>vespa</em></a> RPM package.</li>
+  <li>Download <a href="https://search.maven.org/artifact/com.yahoo.vespa/vespa-feed-client-cli">
+    vespa-feed-client <strong>zip</strong> artifact</a> from Maven Central.</li>
 </ul>
-
+<p>Download example:</p>
+<pre>
+$ F_REPO="https://repo1.maven.org/maven2/com/yahoo/vespa/vespa-feed-client-cli" && \
+  F_VER=$(curl -Ss "${F_REPO}/maven-metadata.xml" | sed -n 's/.*&lt;release&gt;\(.*\)<.*>/\1/p') && \
```
Comment (jonmv) [inline en/vespa-feed-client.html:38]: Forgot to XML-quote the second `<>` pair, though ...

Comment (kkraune) [reply en/vespa-feed-client.html:38]: argh. #2261  to fix that. thanks!

**Pull request merged**